<map id="com.mysql.jdbc.DatabaseMetaData.getInstance" name="com.mysql.jdbc.DatabaseMetaData.getInstance">
<area shape="rect" id="node2" href="$classcom_1_1mysql_1_1jdbc_1_1_database_meta_data.html#a754255faffa1310c4d9507733583ae72" title="com.mysql.jdbc.DatabaseMeta\lData.DatabaseMetaData" alt="" coords="256,41,459,83"/>
<area shape="rect" id="node8" href="$classcom_1_1mysql_1_1jdbc_1_1_util.html#ae60ce70ba2e8e536e773f4abab83c4c1" title="com.mysql.jdbc.Util.handle\lNewInstance" alt="" coords="268,107,447,148"/>
<area shape="rect" id="node3" href="$classcom_1_1mysql_1_1jdbc_1_1_database_meta_data.html#a1a0f564a2d0f6c011143f31f9a0128ef" title="com.mysql.jdbc.DatabaseMeta\lData.getIdentifierQuoteString" alt="" coords="516,8,719,49"/>
<area shape="rect" id="node5" href="$classcom_1_1mysql_1_1jdbc_1_1_assertion_failed_exception.html#a2477a2df55a2af9c8e127fc9a8a0fc34" title="com.mysql.jdbc.Assertion\lFailedException.shouldNotHappen" alt="" coords="507,75,728,116"/>
<area shape="rect" id="node4" href="$interfacecom_1_1mysql_1_1jdbc_1_1_connection.html#a16e68907781a09083ff1afda25dff502" title="com.mysql.jdbc.Connection.supports\lQuotedIdentifiers" alt="" coords="776,5,1013,47"/>
<area shape="rect" id="node6" href="$classcom_1_1mysql_1_1jdbc_1_1_assertion_failed_exception.html#a4e3aad4529bc6d57c1c58d52defe4cb8" title="com.mysql.jdbc.Assertion\lFailedException.AssertionFailed\lException" alt="" coords="790,71,999,127"/>
<area shape="rect" id="node7" href="$classcom_1_1mysql_1_1jdbc_1_1_messages.html#ad6d3160126b07ad0d56d97250bed8412" title="com.mysql.jdbc.Messages.get\lString" alt="" coords="1061,79,1261,120"/>
</map>
